<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResidueListWorkPanel</class>
 <widget class="QWidget" name="ResidueListWorkPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Residue List Editor</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> QPushButton {
     border: 1px solid #dadbde;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
     border-radius: 2px;
     min-width: 34px;
     min-height: 24px;
 }

 QPushButton:pressed {
     border: 2px solid #777777;
 }

 QPushButton:checked {
     border: 2px solid #777777;
 }</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>3</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTabWidget" name="mainTW">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="buildTab">
       <attribute name="title">
        <string>Build</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QPushButton" name="editResiduesPB">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string>Edit residues</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="mergeResiduesPB">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string>Merge residues</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteResiduesPB">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string>Delete residues</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteEmptyStructuresPB">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string>Delete empty</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="orderTab">
       <attribute name="title">
        <string>Order</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="margin">
         <number>3</number>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Sequence index</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>3</number>
           </property>
           <property name="margin">
            <number>3</number>
           </property>
           <item>
            <widget class="QPushButton" name="seqIndxByOrderPB">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>by order</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="seqIndxByReverseOrderPB">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>by reverse order</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="seqIndxByChainsPB">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>by chains</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QCheckBox" name="automaticallyReorderCB">
               <property name="text">
                <string>Automatically reorder</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="reorderBySeqIndxPB">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string>Reorder by sequence index</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Serial index</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <property name="spacing">
            <number>3</number>
           </property>
           <property name="margin">
            <number>3</number>
           </property>
           <item>
            <widget class="QPushButton" name="serIndxBySeqLocPB">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>by seq and local indexes</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QCheckBox" name="automaticallyUpdateSerIndxCB">
               <property name="text">
                <string>Automatically update</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="chainsTab">
       <attribute name="title">
        <string>Chains</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QPushButton" name="setChainNamesPB">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string>Set names of chains</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>241</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="showTab">
       <attribute name="title">
        <string>Show</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="margin">
         <number>2</number>
        </property>
        <item>
         <widget class="QCheckBox" name="highlightResiduesCB">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string>Highlight selected residues</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="margin">
        <number>3</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="margin">
          <number>2</number>
         </property>
         <item>
          <widget class="QLabel" name="label1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Structure:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QObjectLinkWidget" name="structureW" native="true"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget_2">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>Editor</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <property name="spacing">
            <number>3</number>
           </property>
           <property name="margin">
            <number>3</number>
           </property>
           <item>
            <widget class="QTableWidget" name="editTW">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAsNeeded</enum>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="verticalScrollMode">
              <enum>QAbstractItemView::ScrollPerPixel</enum>
             </property>
             <property name="horizontalScrollMode">
              <enum>QAbstractItemView::ScrollPerPixel</enum>
             </property>
             <property name="showGrid">
              <bool>false</bool>
             </property>
             <property name="gridStyle">
              <enum>Qt::NoPen</enum>
             </property>
             <property name="sortingEnabled">
              <bool>false</bool>
             </property>
             <attribute name="horizontalHeaderMinimumSectionSize">
              <number>0</number>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderMinimumSectionSize">
              <number>0</number>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_6">
          <attribute name="title">
           <string>Residues</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <property name="margin">
            <number>2</number>
           </property>
           <item>
            <widget class="QTreeView" name="residuesTV">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="rootIsDecorated">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_5">
          <attribute name="title">
           <string>Atoms</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="margin">
            <number>2</number>
           </property>
           <item>
            <widget class="QTreeView" name="atomsTV">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="rootIsDecorated">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Number of residues:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="numOfResiduesLE">
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QObjectLinkWidget</class>
   <extends>QWidget</extends>
   <header location="global">ObjectLinkWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
